bootstrapvz/plugins/minimize_size/tasks/shrink.py

```diff
@@ -8,6 +8,20 @@
 import os
 
 
+def get_shrink_type(plugins):
```

Comment:
The tasks module should really only contain actual `Task`s. I recommend putting this in `__init__`